main.js

```diff
@@ -75,13 +85,25 @@ const run = async () => {
     }
 
     core.debug('Signing event..')
-    const eventObject = finalizeEvent(rawEvent, keyRaw)
+    const event = finalizeEvent(rawEvent, keyRaw)
 
     core.debug('Validating event..')
-    verifyEvent(eventObject) || die('event is not valid')
+    verifyEvent(event) || die('event is not valid')
     
     core.debug('Sending event..', dry ? '(dry-run enabled: event will not be sent)' : '')
-    const event = dry ? await sendEventDry(relay, eventObject) : await sendEvent(relay, eventObject)
+
+    let errors = []
+    for (const relay of relays) {
+      try {
+        dry ? await sendEventDry(relay, event) : await sendEvent(relay, event)
+      } catch (e) {
+        core.warning(e)
+        errors = [...errors, e]
+      }
+    }
```

Comment:
What is the reason for not using `SimplePool`?

Reply:
Did not even consider. Now that I looked at it, the only real benefit is less code (which is good!) but less flexibility (e.g. more comple dry-run code). Maybe in a future release if more complex features are needed.